src/metpy/plots/declarative.py

```diff
@@ -1831,3 +1834,267 @@ def _build(self):
     def copy(self):
         """Return a copy of the plot."""
         return copy.copy(self)
+
+
+@exporter.export
+class PlotGeometry(HasTraits):
+    """Plot collections of Shapely objects and customize their appearance."""
+
+    import shapely.geometry as shp
```

Comment:
This will always be executed when the class body is run. I think the only way around this is by putting this inside the appropriate methods.